pkg/backends/calico/routes.go

```diff
@@ -101,9 +100,6 @@ func (rg *routeGenerator) Start(routeString string) {
 		if len(cidr) == 0 {
```

Comment:
In the new model, only a single CIDR is supported.

So, we should remove this loop and just parse the CIDR if it is present.